### PR TITLE
Update feishu from 4.11.14 to 5.0.9

### PR DESCRIPTION
**Pull request by chunheisiu**
Created with `brew bump-cask-pr`.

### CODE REVIEW

File: Casks/feishu.rb [hunk 1]
```diff
@@ -2,8 +2,8 @@
   arch = Hardware::CPU.intel? ? "x64" : "arm64"
 
   if Hardware::CPU.intel?
-    version "4.11.14,57b437"
-    sha256 "d083a7d0cf53555687ea63db0608396ad8d3631f5ce6bfac9772653cbf20c474"
+    version "5.0.9,28b420"
+    sha256 "7b25094338f0bf000f8751038a457bc8586dc35738b286b611c4f2ad21063ff2"
   else
     version "4.11.14,204151"
     sha256 "6a66b97a0393402481e44d92d07758d460064e8203c046481400493ac6841769"
```
Comment (Yuiham) [inline Casks/feishu.rb]:
```suggestion
    version "5.0.9,fd8d60"
    sha256 "f3a4711ddf7c95c63c3ec45842c7726856da4ac6619653a7bbd9f62644711b38"
```

Comment (Yuiham) [reply Casks/feishu.rb:9]: @chunheisiu Add the missing arm64 version.

**Review by Yuiham — COMMENTED** [review]
Please add the arm64 version update.

**Review by SMillerDev — APPROVED** [review]

**Pull request merged**